traffic_ops/app/db/migrations/20191004000000_add_server_capability.sql

```diff
@@ -0,0 +1,27 @@
+/*
```

Comment:
I know you're already aware, but just a reminder to delete this file and rebase onto master now that #3966 has been merged.